command/sub8_launch/launch/sub8.launch

```diff
@@ -1,4 +1,5 @@
 <launch>
+  <rosparam param="/autonomous">False</rosparam>
```

Comment:
We can override the default false value from the command line before launching shore control.